great_expectations/render/v3/notebook_assets/suite_edit/header.py.j2

```diff
@@ -17,6 +17,7 @@ context = ge.data_context.DataContext()
 {% if batch_request %}
 # Note that if you modify this batch request, you may save the new version as a .json file
 #  to pass in later via the --batch-request option
+# "limit" refers to the number of batches returned
```

Comment:
Thank you for adding this! Could we add a bit more about which batches are returned (I'm assuming it's the n most recent, rather than the first n)?

Reply:
It looks like it actually returns the first n batches. `batch_request` `limit` is passed to `data_connector_query` and `selected_batch_definitions` are [sliced with [: self.limit]](https://github.com/great-expectations/great_expectations/blob/dc41fe97cab7db5879061727ff6b89ad9fbccf33/great_expectations/datasource/data_connector/batch_filter.py#L202). Is this the expected behavior? You could always use negative numbers (like we do with `index`) to get the n most recent batches.